plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/newapp/ApplicationSourceListPage.java

```diff
@@ -262,7 +262,10 @@ private SelectionListener onBrowseProjects() {
 			@Override
 			public void widgetSelected(SelectionEvent e) {
 				SelectExistingProjectDialog dialog = 
-						new SelectExistingProjectDialog(model.getEclipseProject() == null? null: model.getEclipseProject().getName(), getShell());
+					new SelectExistingProjectDialog(model.getEclipseProject() == null
```

Comment:
fix works but to me the message is misleading when "Change selected project ..." is displayed. It would be more clear imho it would state "Currently project ... is selected. You may freely choose a different one"